web/templates/popovers/send_later_popover.hbs

```diff
@@ -53,4 +53,10 @@
     <li class="drafts-item-in-popover">
         <a href="#drafts">{{t "Drafts"}} <span class="compose-drafts-count-container">(<span class="compose-drafts-count"></span>)</span></a>
     </li>
+    {{#if show_compose_new_message}}
+    <hr />
+    <li>
+        <a class="compose_new_message" tabindex=="0">{{t "Save draft and start a new message"}}</a>
```

Comment:
This should be `e" }}`

Reply:
Hmm, it's less uniform than I'd have thought:
```
$ git grep '{{t "' | grep '"}}' | wc
    104     719   11614
$ git grep '{{t "' | grep '" }}' | wc
    531    4138   68899
```

May be worth discussing in #frontend, but we could also decide we're likely to reformat these templates eventually.

Reply:
Yeah I generally prefer to only have formatting rules like this that are enforced with a linter.